ast/misc.go

```diff
@@ -2555,6 +2556,18 @@ func (ht *HintTable) Restore(ctx *format.RestoreCtx) {
 		ctx.WriteKeyWord("@")
 		ctx.WriteName(ht.QBName.String())
 	}
+	if len(ht.PartitionList) > 0 {
+		ctx.WriteKeyWord("(")
+		for i, p := range ht.PartitionList {
+			if i > 0 {
+				ctx.WriteKeyWord(" ")
+			}
+			ctx.WriteKeyWord("PARTITION[")
```

Comment:
```suggestion
			ctx.WriteKeyWord("PARTITION")
			ctx.WritePlain("[")
```

why use square bracket here 🤔